api/nodes/serializers.py

```diff
@@ -872,7 +876,8 @@ def get_absolute_url(self, obj):
         )
 
     def get_unregistered_contributor(self, obj):
-        unclaimed_records = obj.user.unclaimed_records.get(obj.node._id, None)
+        # SerializerMethodField works for both Node and Preprint contributors
+        unclaimed_records = obj.user.unclaimed_records.get(getattr(obj, 'preprint', getattr(obj, 'node', None))._id, None)
```

Comment:
Does this work? `obj.user.unclaimed_records.get` only accepts a single parameter and it looks like `._id` could be called on `None`